src/elastic-training/index.md

```diff
@@ -0,0 +1,76 @@
+# Elastic Training
+
+## Introduction
+
+By applying [TorchElastic](https://github.com/pytorch/pytorch/tree/v1.9.0/torch/distributed/elastic), bagua can do elastic training. We usually use the capabilities of Elastic Training to solve the following two problems:
```

Comment:
to support the following two types of jobs

Reply:
DONE